### PR TITLE
Add closing dates translations

### PR DESCRIPTION
**Pull request by mangirdasvinted**
@vinted/magic-backend 

https://gist.github.com/zverseckas/10a5fef2d6f25c01007e06694cdadb31#chronopost

### CODE REVIEW

**Comment by mangirdasvinted** [comment]
@vinted/stardust

File: lib/chronopost/parcel_shops/search/translations.rb [hunk 1]
```diff
@@ -30,8 +30,12 @@ module Translations
           coord_geolocalisation_longitude: :longitude,
           url_google_maps: :google_maps_url,
           acces_personne_mobilite_reduite: :reduced_mobility_access,
+          calendar_de_debut: :begin,
```
Comment (paulius-pilkauskas) [inline lib/chronopost/parcel_shops/search/translations.rb]: I think `starts_at` will be more explicit

File: lib/chronopost/parcel_shops/search/translations.rb [hunk 2]
```diff
@@ -30,8 +30,12 @@ module Translations
...
+          calendar_de_fin: :end,
```
Comment (paulius-pilkauskas) [inline lib/chronopost/parcel_shops/search/translations.rb]: I think `ends_at` will be more explicit

File: lib/chronopost/parcel_shops/search/translations.rb (hunk 1 above)
Comment (mangirdasvinted) [reply lib/chronopost/parcel_shops/search/translations.rb]: o pabaiga tada: "ends_at"?

**Pull request merged**